Fixes Issue 35286 Firefox inconsistent scrolling

sorbaugh opened the pull request:
* Resolves: #35286

## Summary

URL: https://github.com/nextcloud/server/issues/35286

Adding overflow-anchor: none to #app-content element within files.scss and re-minified files.css and merged.css.

Tested on Firefox 111.0 (64-Bit)
## TODO

- [ ] ...

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

Commit:
Fixes Issue 35286 Firefox inconsistent scrolling

URL: https://github.com/nextcloud/server/issues/35286

Adding overflow-anchor: none to #app-content element within files.scss and re-minified files.css and merged.css.

Tested on Firefox 111.0 (64-Bit)

Signed-off-by: Stephan <stephan_orbaugh@outlook.de>

artonge commented on apps/files/css/files.scss:
```diff
@@ -114,6 +114,7 @@
 	// force  the width to be the full width to not go bigger than the screen
 	// flex will grow for the mobile view if necessary
 	width: calc(100% - 300px);
+	overflow-anchor: none;
```
Can you add a comment explaining why this is needed ?

artonge replied:
:blue_heart:

artonge commented:
Can you also run `npm ci` and `npm run build`, and also amend your commit with a sign-off? Otherwise, CI will not pass.

sorbaugh commented:
Hello! I ran npm run build as requested, which altered 98 files in the /dist directory of my fork. The purpose of these changes seem to be mainly for cache-busting / adding a new version number to the source-mapping url. 

Since I'm new to contributing in nextcloud and the main change is so small, it feels kind of overkill to commit this many files. So before I commit I wanted to ask if this is ok?

Best regards

artonge commented:
> Since I'm new to contributing in nextcloud and the main change is so small, it feels kind of overkill to commit this many files. So before I commit I wanted to ask if this is ok?

Indeed, not the best, but this is how we currently do things :)

Commit:
Fixes Issue 35286 Firefox inconsistent scrolling

URL: https://github.com/nextcloud/server/issues/35286

Added comment clarifying the purpose of the previous change.
Adding /dist files after npm run build

Tested on Firefox 111.0 (64-Bit)

Signed-off-by: Stephan Orbaugh <stephan_orbaugh@hotmail.com>
Signed-off-by: Stephan <stephan_orbaugh@outlook.de>

sorbaugh commented:
Ok I assume the conflicts with the /dist directory were to be expected if npm run build is run frequently :)

If everything else is alright and ready for merge, I can do a rebase.
